src/Config.php

```diff
@@ -88,9 +88,8 @@ public function getCustomFixers(): array
      */
     public function getFinder(): iterable
     {
-        if (null === $this->finder) {
-            $this->finder = new Finder();
-        }
+        // to master line
```

Comment:
What does this comment mean?

Reply:
buahaha, good eye, thanks. I will remove it: https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7987

I was having gazilion of "draft" commets on my local copy of 7777 and was drafting with comment which change shall go to which PR/target. ([examples](https://github.com/Wirone/PHP-CS-Fixer/pull/5))